### PR TITLE
Release 0.4

### PR DESCRIPTION
**Pull request by frapac**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ notifications:
 script:
  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
  - julia -e 'Pkg.clone(pwd()); Pkg.build("StochDynamicProgramming"); Pkg.test("StochDynamicProgramming"; coverage=true)'
+before_script:
+  - julia -e 'Pkg.clone("https://github.com/StructJuMP/StructJuMP.jl.git")'
+  - julia -e 'Pkg.clone("https://github.com/blegat/StochasticDualDynamicProgramming.jl")'
```
Comment (odow) [inline .travis.yml]: https://github.com/blegat/StructDualDynProg.jl

File: README.md [hunk 2]
```diff
@@ -54,10 +54,14 @@ control strategies.
...
 ## Installation
-Installing StochDynamicProgramming is an easy process. Open Julia and enter
+Installing StochDynamicProgramming is an easy process.
+Currently, the package depends upon `StochasticDualDynamicProgramming.jl`, which is not
+yet registered in Julia's METADATA. To install the package,
+open Julia and enter
 
 ```julia
 julia> Pkg.update()
+julia> Pkg.clone("https://github.com/blegat/StochasticDualDynamicProgramming.jl")
```
Comment (odow) [inline README.md]: https://github.com/blegat/StructDualDynProg.jl

File: .travis.yml (hunk 1 above)
Comment (blegat) [reply .travis.yml]: StructJuMP does not need to be cloned anymore

**Review by leclere — APPROVED** [review]
Agree to merge

**Pull request merged**